tests/v2/actions/cli/cli.go

```diff
@@ -109,31 +109,3 @@ func DeleteNamespaces(client *ranchercli.Client, namespaceName, projectName stri
 
 	return nil
 }
-
-// CreateCatalogs will create and delete catalogs in the specified cluster.
-func CreateCatalogs(client *ranchercli.Client, catalogName string) error {
-	logrus.Infof("Creating a catalog...")
-	err := client.ExecuteCommand(rancher, catalog, "add", catalogName, "https://git.rancher.io/helm3-charts")
```

Comment:
Shouldn't we use another URL instead of deleting the test? Or is the test no longer relevant?

Reply:
Sounds good then!